packages/component-library/src/Stack.tsx

```diff
@@ -46,6 +46,11 @@ type StackProps = ComponentProps<typeof View> & {
   spacing?: number;
   debug?: boolean;
 };
+
+/**
+ * @deprecated This is a terrible component. Please prefer using the `SpaceBetween` component instead.
```

Comment:
@coderabbitai please create an issue to track the removal of the `Stack` component. The ticket should have a todo list of all the places where this component is currently used.